README.md

```diff
@@ -1 +1,47 @@
+=======
+Bitcoin Message Verification and Signing for Bitcore
+=======
+
+[![NPM Package](https://img.shields.io/npm/v/bitcore-message.svg?style=flat-square)](https://www.npmjs.org/package/bitcore-message)
+[![Build Status](https://img.shields.io/travis/bitpay/bitcore-message.svg?branch=master&style=flat-square)](https://travis-ci.org/bitpay/bitcore-message)
+[![Coverage Status](https://img.shields.io/coveralls/bitpay/bitcore-message.svg?style=flat-square)](https://coveralls.io/r/bitpay/bitcore-message?branch=master)
+
+bitcore-message adds support for verifying and signing bitcoin messages in [Node.js](http://nodejs.org/) and web browsers.
+
+See [the main bitcore repo](https://github.com/bitpay/bitcore) for more information.
+
+## Getting Started
+
+```sh
+npm install bitcore-message
+```
+
+```sh
+bower install bitcore-message
+```
+
+To sign a message:
+
+```javascript
+var privateKey = PrivateKey.fromWIF('cPBn5A4ikZvBTQ8D7NnvHZYCAxzDZ5Z2TSGW2LkyPiLxqYaJPBW4');
+var signature = Message('hello, world').sign(privateKey);
+```
+
+To verify a message:
+
+```javascript
+var address = 'n1ZCYg9YXtB5XCZazLxSmPDa8iwJRZHhGx';
+var signature = 'H/DIn8uA1scAuKLlCx+/9LnAcJtwQQ0PmcPrJUq90aboLv3fH5fFvY+vmbfOSFEtGarznYli6ShPr9RXwY9UrIY=';
+var verified = Message('hello, world').verify(address, signature);
+```
+
+## Contributing
+
+See [CONTRIBUTING.md](https://github.com/bitpay/bitcore) on the main bitcore repo for information about how to contribute.
```

Comment:
Should this link directly to the CONTRIBUTING.md file instead of the repository homepage? I've noticed the other bitcore-* modules link to the homepage as well.